src/app/static/src/views/Accessibility.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div class="accessibility">
+    <h1>Accessibility</h1>
+    <p>This statement applies to content published on https://comet.dide.ic.ac.uk</p>
+    <p>
+      We want this website to be as accessible as possible. For example, that means you should
+      be able to:
+    </p>
+    <ul>
+      <li>Resize your window with content being reformatted appropriately</li>
+      <li>Adjust your text size without the site becoming less usable</li>
+      <li>
+        Read text easily due to sufficient contrast between foreground and background elements
+      </li>
+    </ul>
+    <p>
+      <a href="https://mcmw.abilitynet.org.uk/">AbilityNet</a> has advice on making your device easier to
+      use if you have a disability.
+    </p>
+    <h2>How accessible the website is</h2>
+    <p>Parts of this website may not be fully accessible. For example:</p>
+    <ul>
+      <li>Pages may not have their language attribute defined</li>
```

Comment:
This particular issue is trivial to fix so lets just do that here! Just need to add `lang="en"` to the html tags

Reply:
Good point - done. Though I didn't feel we could then claim that the app was completely accessible so I've just put in a bit about interpreting data. Does that look OK?

Reply:
Yeah that sounds good to me, it's a more honest account of how we fail to be accessible also!